spec/spec_helper.rb

```diff
@@ -123,8 +123,8 @@ def correct_forward_diff
       ],
       [
         ['-',  8, 'n'],
-        ['-',  9, 'p'],
         ['+',  9, 'r'],
+        ['-',  9, 'p'],
```

Comment:
Updating the order due to [this discussion](https://github.com/halostatue/diff-lcs/issues/77#issuecomment-998836544)